android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/FxSuggestSuggestionProvider.kt

```diff
@@ -47,6 +47,7 @@ class FxSuggestSuggestionProvider(
         if (text.isEmpty()) {
             emptyList()
         } else {
+            val maxNumOfFirefoxSuggestions = 1
```

Comment:
Could you make this a `const val`, please? [Here are some examples of that](https://github.com/mozilla-mobile/firefox-android/blob/068f97524c55bbb65944ba09fc158a11ae089723/fenix/app/src/main/java/org/mozilla/fenix/compose/TabCounter.kt#L33-L41).